cleaning_services/__manifest__.py

```diff
@@ -0,0 +1,69 @@
+{
+    'name': 'Cleaning Services',
+    'version': '1.0',
+    'category': 'Services',
+    'description': '',
+    'author': 'Odoo S.A.',
+    'depends': [
+        'account_online_synchronization',
+        'crm_sale_subscription',
+        'hr_skills',
+        'industry_fsm_sale_report',
+        'knowledge',
+        'project_purchase',
+        'project_sale_subscription',
+        'web_studio',
+    ],
+    'data': [
+        'data/res_config_settings.xml',
+        'data/ir_model.xml',
+        'data/ir_model_fields.xml',
+        'data/ir_ui_view.xml',
+        'data/ir_actions_act_window.xml',
+        'data/ir_model_access.xml',
+        'data/ir_rule.xml',
```

Comment:
To me, all those files are default records created at the installation of industry_fsm. Could you try by removing it and see how it impacts the db?
```suggestion
```